scripts/variantstore/wdl/GvsMergeScatteredCallsetShards.wdl

```diff
@@ -0,0 +1,123 @@
+version 1.0
+
+workflow GvsMergeScatteredCallsetShards {
+  input {
+    String input_vcfs_directory_plus_prefix
+    Int num_shards = 500
+    String output_vcf_base_name
+    String output_directory
+    String? service_account_json_path
+  }
+
+  call GenerateOrderedPaths as VCFpaths {
+    input:
+      root_path = input_vcfs_directory_plus_prefix,
+      num_files = num_shards,
+      path_suffix = ".vcf.gz"
+  }
+
+  call GenerateOrderedPaths as VCFIndexpaths {
```

Comment:
did you confirm that you need indexes in order to run this tool?  I have a feeling you don't and then you could eliminate this whole path

Reply:
looks like you were right — removed and thanks!